apps/devportal/data/markdown/pages/learn/getting-started/xm-cloud/tutorials/setup-xm-cloud.md

```diff
@@ -0,0 +1,183 @@
+ 
+
+Now, you need to set the API key within .env.local. The API key should be already available within your XM Cloud instance.  
+ 
+3. Now you need to set the Context ID in the .env.local file. Therefore  
+ * Go to the XM Cloud deploy app 
+ * Click on the name of the project you created e.g. XM Cloud Tutorial Series 
+ * Copy the Context ID (Preview) 	 
+![Get Context ID from XM Cloud Deploy](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8ecc65168a504cc188ee84cd8951244b?v=a04033a5) 
+4. Add the Context ID to the `.env.local` file 
+![Copy Context ID into .env file](https://sitecorecontenthub.stylelabs.cloud/api/public/content/e29d1bb0c6a448efb63dce32af047d49?v=2edbe4d1) 
+5. Add the Sitename to the `.env.local` which should be “Company Dev”
+![Set Site Name in .env file](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8fbd4505019948f08bfba8c8be4d4043?v=79a1d301) 
+6. In your terminal navigate to src/sxastarter (your app folder) and run `npm install`
+![In Command Line Interface - Run npm install](https://sitecorecontenthub.stylelabs.cloud/api/public/content/38c87be1d50c4b40abab26556102c4f6?v=474160b6) 
+7. Now run npm run start:connected
+![Start the app locally](https://sitecorecontenthub.stylelabs.cloud/api/public/content/ad26063be71c4ae6b9e62e2579948db3?v=1a82e9cc) 
+8. Call your app in the browser http://localhost:3000
+![Check Website running locally](https://sitecorecontenthub.stylelabs.cloud/api/public/content/358a02664690465289d70fe4c9280eae?v=7c0165d9) 
+ 
+#### Verifying the connection  
+
+ 
+
+Let’s change some content to verify you are in fact connected against the preview endpoint. 
+1. Navigate back to your site in XM Cloud. 
+![XM Cloud Dashboard - Open Pages Editor](https://sitecorecontenthub.stylelabs.cloud/api/public/content/8da9bc8363914bbda2b8b4a67e0cd31e?v=00d2e820) 
+2. Click on the tile that says Company Dev. This will open Pages new WYSIWYG Page Editing experience. When opening for the first time, this might take a while.  
+3. Change the headline to whatever you like. In this case, I’ll add 'in town' to the homepage tagline. 
+![Pages Editor - Change Headline inline](https://sitecorecontenthub.stylelabs.cloud/api/public/content/05ac5bbde7244c809d987106b9226fd8?v=6b47e626) 
+4. Your changes are autosaved.  
+5. Switch back to the browser tab where you are viewing the Next.js app on localhost:3000 
+6. Reload the page, and see the changes appear in your local app.  
+![Validate changes in local running app](https://sitecorecontenthub.stylelabs.cloud/api/public/content/6bf98e5e22fe4c689898fcea34a03b59?v=756f5d1a) 
+
+**Congratulations!** You have successfully set up your dev environment with XM Cloud and created your first webpage!  To learn more about XM Cloud and continue on to next steps, check out our next blog post in this series!
```

Comment:
I think we want the formatting here to be more emphasized for this whole thing. Maybe a card?

We should also have related resources at the bottom as columns of cards.

Reply:
I have added a 'Success' Alert in to handle the emphasis for both sections of the tutorial where we do "Congratulations!" messages.